src/Taxonomies/Taxonomy.php

```diff
@@ -356,6 +356,22 @@ public function previewTargets($targets = null)
             ->args(func_get_args());
     }
 
+    public function extraPreviewTargets($targets = null)
+    {
+        if (! $targets) {
+            return \Statamic\Facades\Taxonomy::extraPreviewTargets($this->handle);
+        }
+
+        \Statamic\Facades\Taxonomy::addExtraPreviewTargets($this->handle, $targets);
+
+        return $this;
+    }
+
+    public function previewTargets()
```

Comment:
`previewTargets` should remain as the getter (gets all base+extra targets) and setter (sets the base targets).